compiler/noirc_evaluator/src/brillig/brillig_gen/brillig_black_box.rs

```diff
@@ -194,6 +181,12 @@ pub(crate) fn convert_black_box_call(
         BlackBoxFunc::RecursiveAggregation => unimplemented!(
             "ICE: `BlackBoxFunc::RecursiveAggregation` is not implemented by the Brillig VM"
         ),
+        BlackBoxFunc::Blake3 => {
+            unimplemented!("ICE: `BlackBoxFunc::Blake3` is not implemented by the Brillig VM")
+        }
+        BlackBoxFunc::Keccakf1600 => {
+            unimplemented!("ICE: `BlackBoxFunc::Keccakf1600` is not implemented by the Brillig VM")
+        }
```

Comment:
We should open an issue to implement these in the brillig VM. We have rust solver for blake3 so it should be easy, need to check for the keccak permutation.